src/components/screens/monitor/delegates/overview.js

```diff
@@ -0,0 +1,152 @@
+// istanbul ignore file
+import React from 'react';
+import moment from 'moment';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxEmptyState from '../../../toolbox/box/emptyState';
+import Chart from '../../../toolbox/charts';
+import { typeLine, typeDoughnut } from '../../../../constants/chartConstants';
+import styles from './overview.css';
+
+const Overview = ({
+  chartActiveAndStandby,
+  chartDelegatesForging,
+  chartRegisteredDelegates,
+  delegatesForgedLabels,
+  t,
+}) => {
+  const getAmountOfDelegatesInTime = () => {
+    const totalDelegates = chartActiveAndStandby.data;
+    const final = [totalDelegates];
+    chartRegisteredDelegates.data
+      .map(delegate => (delegate.y))
+      .reduce((amountOfDelegates, amountOfDelegatesByMonth) => {
+        final.unshift(amountOfDelegates - amountOfDelegatesByMonth);
+        return amountOfDelegates - amountOfDelegatesByMonth;
+      }, totalDelegates);
+
+    return final;
+  };
+
+  const getAmountOfDelegatesLabels = () => {
+    const labels = chartRegisteredDelegates.data.map(date => moment(date.x).format('MMM YY'));
```

Comment:
You just iterated over this array in the paretn component where you retrieved the data, why didn't you format it right away? This iteration is redundant.

Reply:
I fixed too